packages/strapi-helper-plugin/lib/src/components/InputsIndex/index.js

```diff
@@ -0,0 +1,36 @@
+/**
+ *
+ * InputsIndex references all the input with errors available
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// Design
+import InputNumberWithErrors from 'components/InputNumberWithErrors';
+import InputTextWithErrors from 'components/InputTextWithErrors';
+import InputToggleWithErrors from 'components/InputToggleWithErrors';
+
+const inputs = {
+  number: InputNumberWithErrors,
+  string: InputTextWithErrors,
+  text: InputTextWithErrors,
+  toggle: InputToggleWithErrors,
+};
+
+function InputsIndex(props) {
+  const Input = inputs[props.type] || <div />;
```

Comment:
Should we display an error or a warning message and doesn't return an empty div?

Reply:
See https://github.com/strapi/strapi/pull/564